dns/push.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
are we actually using bash isms?

Reply:
Removed 'function push ()' at https://github.com/kubernetes/k8s.io/commit/0d0051b5f809d97d45a19aaa33d95010df94aed5#diff-0a5093ca99f6474ed25e192ae7548a7dR7 replacing with 'push ()' at https://github.com/kubernetes/k8s.io/commit/5adb9a657e32e4ae1ed67ad88bc1c6faf128e265

Didn't look fully, but my dash shell didn't like it.